code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -474,3 +474,40 @@
 	icon_state = "cleave"
 	duration = 6
 
+/obj/effect/temp_visual/eldritch_smoke
+	icon = 'icons/effects/eldritch.dmi'
+	icon_state = "smoke"
+	duration = 10
+
+/obj/effect/proc_holder/spell/targeted/fiery_rebirth
+	name = "Nightwatcher's Rebirth"
+	desc = "Drains nearby alive people that are engulfed in flames. It heals 10 of each damage type per person. If a person is in critical condition it finishes them off."
+	invocation = "GL'RY T' TH' N'GHT'W'TCH'ER"
+	invocation_type = "whisper"
+	clothes_req = FALSE
+	action_background_icon_state = "bg_ecult"
+	range = -1
+	include_user = TRUE
+	charge_max = 600
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "smoke"
+
+/obj/effect/proc_holder/spell/targeted/fiery_rebirth/cast(list/targets, mob/user)
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/human_user = user
+	for(var/mob/living/carbon/target in view(7,user))
+		if(target.stat != DEAD && target.on_fire)
+			//This is essentially a death mark, use this to finish your opponent quicker.
+			if(target.InCritical())
+				target.death()
+			target.adjustFireLoss(20)
+			new /obj/effect/temp_visual/eldritch_smoke(target.drop_location())
+			human_user.ExtinguishMob()
+			human_user.adjustBruteLoss(-10)
+			human_user.adjustFireLoss(-10)
+			human_user.adjustStaminaLoss(-10)
+			human_user.adjustToxLoss(-10)
+			human_user.adjustOxyLoss(-10)
```

Comment:
```suggestion
			human_user.adjustBruteLoss(-10, FALSE)
			human_user.adjustFireLoss(-10, FALSE)
			human_user.adjustStaminaLoss(-10, FALSE)
			human_user.adjustToxLoss(-10, FALSE)
			human_user.adjustOxyLoss(-10)
```
This is a review I did multiple times before, and I'd like for you to understand. See, most procs that do damage also take an argument regarding updating health or not.
Updating health means recalculating all the damages, stat, redrawing the huds and whatnot.
If you are adding several kinds of damages, try to only update health at the end of it. There's no need to update it 5 times here, is it?

Reply:
ah alright i never understood why you do the ,FALSE at the end